src/styles/components/_compare.scss

```diff
@@ -10,7 +10,6 @@
 // =============================================================================
 
 .w-compare {
-  font-size: .875em;
   line-height: 1.5em;
   margin: -16px 0 32px;
```

Comment:
I think you can remove the negative top margin. It seems to look better without it.

Reply:
I left it in because I think it improves the connection between paired compare widgets by reducing the vertical spacing between them, but I can remove it.